src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -265,6 +268,57 @@ class PyMobile:
 class PyMulti:
     __class__: str = None
 
+class PyNineSliceGump:
+    NineSliceGump = None
+    Gump: Gump = None
```

Ruff (0.13.1)

Undefined name `Gump`

(F821)